clang/Util.d

```diff
@@ -80,6 +80,11 @@ mixin template CX ()
     }
 }
 
+string clangVersion()
+{
+    return clang_getClangVersion().toD;
```

Comment:
Is this really working? Last time I tried it, it didn't return different values for different versions I tried. Or I made some mistake.